File: GroupBar_NavigationPane_Customization/GroupBar_NavigationPane_Customization_C#/Form1.cs
using Syncfusion.Windows.Forms;
using Syncfusion.Windows.Forms.Tools;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace groupbartest
{
    public partial class Form1 : MetroForm
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void GroupBar1_MouseClick(object sender, MouseEventArgs e)
        {
            Office2007BlueColors.Default.FloatHighlightButtonColor = Color.Pink;
        }
    }
}

File: GroupBar_NavigationPane_Customization/GroupBar_NavigationPane_Customization_VB/Form1.vb
Imports Syncfusion.Windows.Forms
Imports Syncfusion.Windows.Forms.Tools
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace groupbartest
	Partial Public Class Form1
		Inherits MetroForm

		Public Sub New()
			InitializeComponent()


		End Sub

		Private Sub GroupBar1_MouseClick(ByVal sender As Object, ByVal e As MouseEventArgs)
			Office2007BlueColors.Default.FloatHighlightButtonColor = Color.Pink
		End Sub
	End Class
End Namespace
